models/zeroex/polygon/zeroex_polygon_nft_fills.sql

```diff
@@ -0,0 +1,129 @@
+{{  config(
+        alias='nft_fills',
+        materialized='incremental',
+        partition_by = ['block_date'],
+        unique_key = ['block_date', 'tx_hash', 'evt_index'],
+        on_schema_change='sync_all_columns',     
+        file_format ='delta',
+        incremental_strategy='merge',
+        post_hook='{{ expose_spells(\'["polygon"]\',
+                                "project",
+                                "zeroex",
+                                \'["rantum","bakabhai993", "sui414"]\') }}'
+    )
+}}
+
+{% set zeroex_v4_nft_start_date = '2022-03-01' %}
+
+--sample query on dune v2: https://dune.com/queries/1844568 
+WITH tbl_cte_transaction AS
+(
+    SELECT  evt_block_time
+         , evt_tx_hash
+         , evt_index
+         , maker
+         , taker
+         , matcher
+         , erc721Token      AS nft_address
+         , erc721TokenId    AS nft_id
+         , 'erc721'         AS label
+         , CASE
+                WHEN erc20Token = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'
+                THEN '0x7ceb23fd6bc0add59e62ac25578270cff1b9f619'
+                ELSE erc20Token
+            END             AS price_label
+         , erc20Token       AS token
+         , erc20TokenAmount AS token_amount_raw
+    FROM {{ source ('zeroex_polygon', 'ExchangeProxy_evt_ERC721OrderFilled') }}
+    WHERE 1 = 1 
+        {% if is_incremental() %}
+        AND evt_block_time >= date_trunc('day', now() - interval '1 week')
+        {% endif %}
+        {% if not is_incremental() %}
+        AND evt_block_time >= '{{zeroex_v4_nft_start_date}}'
+        {% endif %}
+
+    UNION ALL
+
+    SELECT  evt_block_time
+            , evt_tx_hash
+            , evt_index
+            , maker
+            , taker
+            , matcher
+            , erc1155Token      AS nft_address
+            , erc1155TokenId    AS nft_id
+            , 'erc1155'         AS label
+            , CASE
+                WHEN erc20Token = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee'
+                THEN '0x7ceb23fd6bc0add59e62ac25578270cff1b9f619'
+                ELSE erc20Token
+                END             AS price_label
+            , erc20Token        AS token
+            , erc20FillAmount   AS token_amount_raw
+    FROM {{ source ('zeroex_polygon', 'ExchangeProxy_evt_ERC1155OrderFilled') }}
```

Comment:
Do we also have information about the number of ERC1155s for the order? 
That might be useful to include.

Reply:
Good catch - I added this to the Polygon model, as well as the Ethereum model in the PR